addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -363,6 +363,9 @@ var SnippetEditor = Widget.extend({
         if (this.$target.data('name') !== undefined) {
             return this.$target.data('name');
         }
+        if (this.$target.is('.o_logo')) {
```

Comment:
I don't think `o_logo` is a class from the web_editor app. You should `include` that method in website I guess